docs/RFPs/Open/IDE_for_ink!_Smart_Contracts.md

```diff
@@ -0,0 +1,22 @@
+# Browser based IDE for ink! Smart Contracts 
+
+* **Status:** Open
+* **Proposer:** [David Hawig](https://github.com/Noc2)
+
+## Project Description :page_facing_up:  
+
+[ink!](https://github.com/paritytech/ink) is a domain-specific language for writing smart contracts in Rust and compiles to Wasm code. You can use it with blockchains built on Substrate.
+
+The goal of this RFP is to find teams that develop or maintain browser based playground for editing, sharing & compiling ink! Smart Contracts. We are hereby specifically looking for a developer or team that wants to continue to work on the [ink! Playground](https://github.com/paritytech/ink-playground), which was originally developed and maintained by Parity. But we are also actively looking for teams that want to implement alternative solutions, like, for example, something based on [http://remix.ethereum.org](https://remix.ethereum.org/). 
+
```

Comment:
The [ink! Playground](https://ink-playground.substrate.io/) previously maintained by Parity currently utilizes Typescript, React, Docker, and [Monaco Editor](https://microsoft.github.io/monaco-editor/). Teams could stick with this tech stack, or choose to build something new if desired. For example, there has been some [discussion](https://forum.parity.io/t/using-codespaces-for-a-workshop-initial-thoughts/1745/4?u=keegan_quigley) around the viability of implementing GitHub Codespaces in combination with GitHub Actions.

Reply:
@semuelle thanks, we could perhaps mention that here.